AdminUi/src/AdminUi/ClientApp/.ci/runChecks.mjs

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env zx
+
+process.env.FORCE_COLOR = 3;
```

Comment:
What's that?

Reply:
zx output is all in the same color. This line preserves the error and information colors in the command line.